static/src/javascripts/projects/common/modules/experiments/ab.js

```diff
@@ -182,7 +182,7 @@ define([
         // Get this browser's mvt test id.
             mvtCookieId = mvtCookie.getMvtValue();
 
-        if (smallestTestId <= mvtCookieId && largestTestId > mvtCookieId) {
```

Comment:
the code that sets the id looks like it sets it correctly in the range 1-1000000 inclusive and does indeed orphan id 1000000 from the server side mvt tests https://github.com/guardian/fastly-edge-cache/blob/d7d52878ae680ef3433351c17f8f5de9c62bd946/theguardiancom/src/main/resources/varnish21/ab-tests.vcl#L27